base/replutil.jl

```diff
@@ -380,6 +380,15 @@ function showerror(io::IO, ex::MethodError)
         end
         print(io, ")")
     end
+    if f_is_function && applicable(start, ex.args) && !method_exists(f, arg_types)
+        if Base.iteratorsize(ex.args) == Base.HasLength()
+            print(io, "\nYou may consider implementing an additional",
```

Comment:
I'd just say ``... implementing the `length` method``. Same below (no need to say "such as", as these are exactly the methods to implement).